lib/factory_bot.rb

```diff
@@ -98,6 +100,8 @@ class << self
 
     deprecate :allow_class_lookup,
```

Comment:
Are we also able to deprecate `FactoryBot.factories`?

Reply:
There is one mention of using this method in the [docs](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#linting-factories), maybe the use case it's not needed?